configs/ann/ann.yml

```
@@ -1,296 +1,306 @@
 Collections:
-- Metadata:
+- Name: ann
+  Metadata:
     Training Data:
     - Cityscapes
     - ADE20K
     - Pascal VOC 2012 + Aug
-  Name: ann
+  Paper:
+    URL: https://arxiv.org/abs/1908.07678
+    Title: Asymmetric Non-local Neural Networks for Semantic Segmentation
+  README: configs/ann/README.md
+  Code:
+    URL: https://github.com/open-mmlab/mmsegmentation/blob/v0.17.0/mmseg/models/decode_heads/ann_head.py#L185
+    Version: v0.17.0
+  Converted From:
+    Weights: ''
+    Code: https://github.com/MendelXu/ANN
 Models:
-- Config: configs/ann/ann_r50-d8_512x1024_40k_cityscapes.py
+- Name: ann_r50-d8_512x1024_40k_cityscapes
   In Collection: ann
   Metadata:
     backbone: R-50-D8
     crop size: (512,1024)
+    lr schd: 40000
     inference time (ms/im):
-    - backend: PyTorch
-      batch size: 1
+    - value: 269.54
       hardware: V100
+      backend: PyTorch
+      batch size: 1
       mode: FP32
       resolution: (512,1024)
-      value: 269.54
-    lr schd: 40000
     memory (GB): 6.0
-  Name: ann_r50-d8_512x1024_40k_cityscapes
   Results:
+    Task: Semantic Segmentation
```

```suggestion
    -Task: Semantic Segmentation
```

Refer to https://github.com/open-mmlab/mmdetection/blob/master/configs/atss/metafile.yml#L36.